models/user.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var User = sequelize.define('User', {
+    name: {
+      type: DataTypes.STRING,
+      validate: {
+         notEmpty: {
+             msg: 'Name cannot be empty'
+         }
+      }
+   },
+    surname: {
+      type: DataTypes.STRING,
+      validate: {
+         notEmpty: {
+             msg: 'Surame cannot be empty'
+         }
+      }
+   },
+    slug: {
+      type: DataTypes.STRING,
+      validate: {
+         notEmpty: {
+            msg: 'Slug cannot be empty'
+         }
+      }
+   },
+    age: {
+      type: DataTypes.INTEGER,
+      validate: {
+         notEmpty: {
+             msg: 'Age cannot be empty'
+         }
+      }
+   },
+    email: {
+      type: DataTypes.STRING,
+      validate: {
+         notEmpty: {
+             msg: 'Email cannot be empty'
+         }
+      }
+   },
+    password: {
+      type: DataTypes.STRING,
+      validate: {
```

Comment:
password can be empty, passwordDigestCannot be empty and this field shouldn't be in the database it should have the type DataTypes.VIRTUAL